debian/enterprise-replicator-executable/include/etc/confluent/docker/log4j.properties.template

```diff
@@ -0,0 +1,15 @@
+
+log4j.rootLogger={{ env["REPLICATOR_LOG4J_ROOT_LOGLEVEL"] | default('INFO') }}, stdout
+
+log4j.appender.stdout=org.apache.log4j.ConsoleAppender
+log4j.appender.stdout.layout=org.apache.log4j.PatternLayout
+log4j.appender.stdout.layout.ConversionPattern=[%d] %p %m (%c)%n
+
+log4j.logger.org.reflections=ERROR
```

Comment:
What about these that are also in AK's log config for Connect?
```
log4j.logger.org.apache.zookeeper=ERROR
log4j.logger.org.I0Itec.zkclient=ERROR
```

Reply:
This is template that we is copied to replicator's log4j above. We need to suppress these warning because we embed the connect worker. Other than this template that will result to replicator's `log4j.properties` there's no other log4j properties file I think.